addon/utils/_private/node-utils.ts

```diff
@@ -0,0 +1,10 @@
+import { PNode } from '@lblod/ember-rdfa-editor';
+
+export function getGroups(node: PNode) {
+  return node.type.spec.group?.split(' ') ?? [];
+}
+
+export function hasGroups(node: PNode, ...groups: string[]) {
+  const nodeGroups = getGroups(node);
+  return groups.every((group) => nodeGroups.includes(group));
+}
```

Comment:
Might be an idea to also support `hasGroup`?
Or support passing just a string too (but then the function name makes less sense and that's probably not javascripty).
As I'm guessing you'd more often check for one specific group than a list of them.

Reply:
yes but that function supports that. You can invoke it by using `hasGroups(node, 'group')` or `hasGroups(node, 'group1', 'group2')`

Reply:
Great, I looked over the spread operator :+1: